unlock-app/src/hooks/useUSDPricing.ts

```diff
@@ -9,6 +9,31 @@ interface USDPricingOptions {
   enabled?: boolean
 }
 
+export const getLockUsdPrice = async ({
```

Comment:
Helper to make sure that we always have the same format for `storage.price` because we return `amount: result?.priceInAmount`